bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -256,13 +256,13 @@ spec:
         serviceAccountName: numaresources-controller-manager
     strategy: deployment
   installModes:
-  - supported: false
+  - supported: true
     type: OwnNamespace
-  - supported: false
+  - supported: true
     type: SingleNamespace
   - supported: false
     type: MultiNamespace
-  - supported: true
+  - supported: false
```

Comment:
yes, this makes sense with the direction of the operator as it we see at the moment.